test/controllers/drivers_controller_test.rb

```diff
@@ -1,175 +1,259 @@
 require "test_helper"
 
 describe DriversController do
-  # Note: If any of these tests have names that conflict with either the requirements or your team's decisions, feel empowered to change the test names. For example, if a given test name says "responds with 404" but your team's decision is to respond with redirect, please change the test name.
+  before(:all) do
+    @test_driver = Driver.create(name:"C14", vin: "12345678901234567", available: true)
```

Comment:
I recommend using `create!` here because it will make it obvious if your `before` fails to create any data:

```suggestion
    @test_driver = Driver.create!(name:"C14", vin: "12345678901234567", available: true)
```